### PR TITLE
Testing: switch to Yakforms + Packaging v2

### PR DESCRIPTION
**Pull request by lapineige**
Upgrade to Yakforms 1.1

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by alexAubin** [comment]
!testme

**Comment by alexAubin** [comment]
Waiting for feedback from isaaac to see if he can still reproduce the app's issue (some module is broken somehow) using this new testing branch

You can test installing this branch using `yunohost app install https://github.com/YunoHost-Apps/framaforms_ynh/tree/testing`

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+packaging_format = 2
+
+id = "framaforms"
+name = "Framaforms"
+description.en = "Create online webforms and surveys"
+description.fr = "Créez des formulaires et questionnaires en ligne"
+
+version = "1.1~ynh1"
```
Comment (alexAubin) [inline manifest.toml:8]:
```suggestion
version = "1.0.3~ynh1"
```

inconsistent version compared to sources below ...

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
> Waiting for feedback from isaaac to see if he can still reproduce the app's issue (some module is broken somehow) using this new testing branch
> 
> You can test installing this branch using `yunohost app install https://github.com/YunoHost-Apps/framaforms_ynh/tree/testing`

Any news ?

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 2]
```diff
@@ -31,64 +17,47 @@ ynh_script_progression --message="Checking version..." --weight=1
...
 #=================================================
-# CREATE DEDICATED USER
+# Framaforms 2 Yakforms Backup
 #=================================================
-ynh_script_progression --message="Making sure dedicated system user exists..." --weight=1
+upgrade_to_yakforms=false
 
-# Create a dedicated user (if not existing)
-ynh_system_user_create --username=$app --home_dir=$final_path
+then
```
Comment (ericgaspar) [inline scripts/upgrade:44]: We are missing a `if` statement here

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 3]
```diff
@@ -9,86 +9,58 @@
...
+# Framaforms 2 Yakforms Backup
 #=================================================
-ynh_script_progression --message="Making sure dedicated system user exists..." --weight=1
 
-# Create a dedicated user (if not existing)
-ynh_system_user_create --username=$app --home_dir=$final_path
+if [ "$upgrade_to_yakforms" = false ]
+then
+	update-alternatives --set php /usr/bin/php$phpversion
+    upgrade_to_yakforms=true
```
Comment (alexAubin) [inline scripts/upgrade:49]: Dunno what was the intent for this variable but we're getting `./upgrade: line 46: upgrade_to_yakforms: unbound variable` @_@

Comment (Salamandar) [reply scripts/upgrade:49]: Or just set it to true and roll with it.

**Comment by Salamandar** [comment]
!testme

**Pull request closed without merging**